src/app/modules/side-menu/components/side-menu.component.ts

```diff
@@ -132,18 +126,10 @@ export class SideMenuComponent implements OnInit, OnDestroy {
         .get(['MENU.ERROR_REPORT_DESCRIPTION', 'MENU.CONTACT_REGOBS_ERROR'])
         .toPromise();
       const appVersion = await this.appVersionService.getAppVersion();
-      const email: EmailComposerOptions = {
-        to: settings.errorEmailAddress,
-        subject:
-          `${translations['MENU.CONTACT_REGOBS_ERROR']}: ${
-            this.platform.is('ios') ? 'ios' : ''
-          }` +
+      const subject = `${translations['MENU.CONTACT_REGOBS_ERROR']}: ${this.platform.is('ios') ? 'ios' : ''}` +
           `${this.platform.is('android') ? 'android' : ''}` +
-          ` ${appVersion.version} ${appVersion.buildNumber} ${appVersion.revision}`,
-        body: translations['MENU.ERROR_REPORT_DESCRIPTION'],
-        isHtml: true
-      };
-      this.emailComposer.open(email);
+          ` ${appVersion.version} ${appVersion.buildNumber} ${appVersion.revision}`;
+      this.fileLoggingService.sendLogsByEmail(subject, translations['MENU.ERROR_REPORT_DESCRIPTION']);
```

Comment:
Jeg lurer på om vi burde fortelle brukeren at vi har lagt ved logger i eposten? F.eks. noe slikt (ved rapportering av feil: 
"Vennligst beskriv feilen. Dersom du har skjermbilder, kan du legge de ved. Vi har lagt ved loggfiler som kan hjelpe oss å finne feilen. Du kan fjerne disse vedleggene om du ikke ønsker å sende inn logger".
Tror vi trenger noe tilsvarende for "Kontakt oss", men teksten blir kanskje litt enklere?

Reply:
Det er forøvrig en feil i den norske teksten for ERROR_REPORT_DESCRIPTION. Det mangler en t i "Vennligst"